tests/api.v2/model/Experiment.test.js

```diff
@@ -172,4 +162,26 @@ describe('model/experiment', () => {
     expect(mockTrx.commit).not.toHaveBeenCalled();
     expect(mockTrx.rollback).toHaveBeenCalled();
   });
+
+  it('addSample works correctly', async () => {
+    mockSqlClient.where.mockImplementationOnce(() => { Promise.resolve(); });
+    mockSqlClient.raw.mockImplementationOnce(() => 'RawSqlCommand');
+
+    await new Experiment().addSample(mockExperimentId, mockSampleId);
+
+    expect(mockSqlClient.update).toHaveBeenCalledWith({ samples_order: 'RawSqlCommand' });
+    expect(mockSqlClient.raw).toHaveBeenCalledWith('samples_order || \'["mockSampleId"]\'::jsonb');
+    expect(mockSqlClient.where).toHaveBeenCalledWith('id', 'mockExperimentId');
+  });
+
+  it('deleteSample works correctly', async () => {
```

Comment:
can we test for deleteSample and addSample failing? What other edge cases can we have?